src/Raven.Server/Integrations/PostgreSQL/Npgsql/version_query.csv

```diff
@@ -0,0 +1,2 @@
+"version"
+"PostgreSQL 13.3, compiled by Visual C++ build 1914, 64-bit"
```

Comment:
Are we providing this as an answer?

Reply:
Yes, it's used as version response: 

https://github.com/ravendb/ravendb/blob/98f362180fb527c000ea90f958cd633b0fb7cb0e/src/Raven.Server/Integrations/PostgreSQL/Npgsql/NpgsqlConfig.cs#L14-L19